style.css

```diff
@@ -0,0 +1,108 @@
+@font-face {
+  font-family: Poppins;
+  src: url("./resources/font/Poppins-Regular.ttf");
+}
+
+@font-face {
+  font-family: Poppins-Bold;
+  src: url("./resources/font/Poppins-Bold.ttf");
+}
+
+@media only screen and (max-width: 600px) {
+  body {
+    font-family: Poppins, sans-serif;
+    font-style: normal;
+    margin: 0;
+  }
+}
+
+html {
+  background: #e5e5e5;
+}
+
+.article {
+  background-image: url("./resources/images/header-shapes\ mobilebackground-Main.png");
+  display: flex;
+  flex-direction: column;
+  min-height: 100vh;
+  background-size: 100%;
+  background-repeat: no-repeat;
+  background-position: 1% 1%;
+  align-items: flex-start;
+  position: absolute;
+  border-bottom-left-radius: 15%;
+  background-color: #fff;
+}
+
+.article-item1 {
+  margin: 15%;
+}
```

Comment:
- [ ] Kindly adjust the margin to the headline section. Your design is skewed to the right while that of the Figma design is skewed to the left. Please adjust the margin and make it identical to the Figma design that you chose. 👍 


| Your Design | Figma Design |
| ------------- | ------------- |
| ![Screenshot_1](https://github.com/Uzair-Manzoor/Portfolio/assets/77106962/5560fe7e-19ed-4136-b1ab-a4bbeb0f7160) | ![Screenshot_2](https://github.com/Uzair-Manzoor/Portfolio/assets/77106962/609e1a7a-35c8-44b0-8bf1-d19a7c5323cf) |